src/lib/containers/synapse_form_wrapper/SynapseForm.tsx

```diff
@@ -893,6 +896,7 @@ export default class SynapseForm extends React.Component<
                   } `}
                 >
                   <Form
+                    noHtml5Validate
```

Comment:
Changing to use the submit button started triggering the HTML5 validation. The component doesn't use it today (the custom validation may be more specific), so I disabled it.